charts/sentry/templates/pgbouncer/pgbouncer-deployment.yaml

```diff
@@ -0,0 +1,65 @@
+{{- if .Values.pgbouncer.enabled }}
```

Comment:
oh, a "proxysql" for postgreSQL... how nice, but I don't want more dependencies please :joy: